conftest.py

```diff
@@ -54,3 +48,14 @@ def exporter(find_free_port, celery_config):
     exporter = Exporter()
     setattr(exporter, "cfg", cfg)
     yield exporter
+
+
+@pytest.fixture
+def caplog(_caplog):
```

Comment:
Taken from loguru project. A very useful and clean way of testing correctness of logs in the project.